src/pages/blog/[slug].astro

```diff
@@ -43,28 +45,30 @@ const { post } = Astro.props;
 ---
 
 <Layout
-  title={post.title}
+  title={"cats"}
```

Comment:
What's this???

Reply:
Honest mistake. Leftover placeholder while debugging

Reply:
Meow it looks good 🐱